lowelldevclub/templates/about.html

```diff
@@ -7,44 +8,55 @@
-    	<a href="#" target="_blank"><i class="fa fa-instagram"></i></a>
-  	</div>
-  	<p><button>Contact</button></p>
-	</div>
-	<div class="card">
-  	<img src="" alt="Luna, Isabel" style="width:100%">
-  	<h1>Luna, Isabel</h1>
-  	<p class="title">Vice President</p>
-  	<p>Harvard University</p>
-  	<div style="margin: 24px 0;">
-    	<a href="#" target="_blank"><i class="fa fa-github"></i></a>
-			<a href="#" target="_blank"><i class="fa fa-linkedin"></i></a>
-    	<a href="#" target="_blank"><i class="fa fa-facebook"></i></a>
-    	<a href="#" target="_blank"><i class="fa fa-instagram"></i></a>
-  	</div>
-  	<p><button>Contact</button></p>
+	<div class="container">
+  	<div class="row">
+			<div class="col">
+				<div class="card">
+  				<img src="static/img/rafael_profile.jpeg" alt="Rafael Cenzano" style="height:200">
+  				<h1>Rafael Cenzano</h1>
+  				<p class="title">Co-President</p>
+  				<p>Python, Bash Scripting, Javascript, Hyper Text Markup Language, Cascading Style Sheets</p>
+  				<div style="margin: 24px 0;">
+    				<a href="https://github.com/RafaelCenzano" target="_blank"><i class="fa fa-github"></i></a>
+    				<a href="https://www.linkedin.com/in/rafael-cenzano/" target="_blank"><i class="fa fa-linkedin"></i></a>
+    				<a href="https://www.facebook.com/profile.php?id=100008046498255" target="_blank"><i class="fa fa-facebook"></i></a>
+    				<a href="https://www.instagram.com/raf.py/" target="_blank"><i class="fa fa-instagram"></i></a>
+  				</div>
+  				<a href="mailto:savagecoder77@gmail.com"><p><button>Contact</button></p></a>
+				</div>
+		  </div>
+			<div class="col">
+				<div class="card">
+  				<img src="" alt="Cappillen Lee" style="width:100%">
+  				<h1>Cappillen Lee</h1>
+  				<p class="title">Co-President</p>
+  				<p>Python, Java, Javascript, Hyper Text Markup Language, Cascading Style Sheets</p>
+  				<div style="margin: 24px 0;">
+    				<a href="#" target="_blank"><i class="fa fa-github"></i></a>
+						<a href="#" target="_blank"><i class="fa fa-linkedin"></i></a>
+    				<a href="#" target="_blank"><i class="fa fa-facebook"></i></a>
+    				<a href="#" target="_blank"><i class="fa fa-instagram"></i></a>
+  				</div>
+  				<p><button>Contact</button></p>
+				</div>
+			</div>
+			<div class="col">
+				<div class="card">
+  				<img src="" alt="Luna, Isabel" style="width:100%">
+  				<h1>Luna, Isabel</h1>
+  				<p class="title">Vice President</p>
+  				<p>Python, Javascript, Hyper Text Markup Language, Cascading Style Sheets</p>
+  				<div style="margin: 24px 0;">
+    				<a href="#" target="_blank"><i class="fa fa-github"></i></a>
+						<a href="#" target="_blank"><i class="fa fa-linkedin"></i></a>
+    				<a href="#" target="_blank"><i class="fa fa-facebook"></i></a>
+    				<a href="#" target="_blank"><i class="fa fa-instagram"></i></a>
+  				</div>
+  				<p><button>Contact</button></p>
+				</div>
+			</div>
```

Comment:
Add the <div style="clear: both"></div> right in between line 58 and 59.

Reply:
Fixed the image problem

Reply:
add what between those lines?